coordinator/src/bin/coordinator.rs

```diff
@@ -189,6 +191,9 @@ async fn main() -> Result<()> {
         }
     });
 
+    let (tx_price_feed, _rx) = broadcast::channel(100);
```

Comment:
Maybe I am mistaken, but the `tx_price_feed` is received in the `websocket.rs` and is useful because with it I can broadcast the updated price to all connected users instead of having to send it to everyone individually.

```
// We subscribe *before* sending the "joined" message, so that we will also
// display it to our client.
let mut rx = state.tx_price_feed.subscribe();
```

cc: @bonomat